packages/cloudreverb/PKGBUILD

```
@@ -0,0 +1,73 @@
+# Maintainer: OSAMC <https://github.com/osam-cologne/archlinux-proaudio>
+# Contributor: Christopher Arndt <aur -at- chrisarndt -dot- de>
+
+_name=CloudReverb
+pkgname=${_name,,}
+_plugin_uri='http://github.com/xunil-cloud/$_name'
```

Should be
```suggestion
_plugin_uri="http://github.com/xunil-cloud/$_name"
```
in order to resolve the variable in this place, right?